designsafe/static/scripts/workspace/services/systems-service.js

```diff
@@ -3,7 +3,7 @@
  * lookup right now, but it will be nicer to go ahead and code against this service
  * rather than having stubs all over the place.
  */
-export function workspaceSystemsService($q, $http) {
+export default function workspaceSystemsService($q, $http) {
```

Comment:
```suggestion
export function workspaceSystemsService($q, $http) {
```
This either needs to be a non-default export, or anything that imports it needs to be updated to remove the squiggly braces.